roles/ffh.dhcp_server/tasks/per_domain.yml

```diff
@@ -1,8 +1,10 @@
-- name: set variables
+---
+
+- name: Set domain-specific variables
   set_fact:
     dhcp_net:
       net: "10.{{ domain.id }}.0.0"
-      mask: "255.255.0.0"
+      cidr: 16
     dhcp_interface: "bat{{ domain.id }}"
     dhcp_range:
```

Comment:
Maybe we should rethink our decision to use decimal-oriented boundaries for our ranges. I would prefer ranges which can simply be represented in a cidr notation.